cheri-c-programming.tex

```diff
@@ -884,7 +884,8 @@ \subsubsection{Out-of-bounds pointers}
 some point the bounds can no longer be represented within the capability.
 As a result, the architecture prohibits manipulations which would produce such
 a capability; in general, attempts to do so will clear the tag, resulting in
-an invalid capability -- leading attempting dereferences to fail in the same
+an invalid capability.
+Dereferencing such a capability will fail in the same
```

Comment:
Possibly should be “Attempting to dereference...”, since arguably the exception prevents the dereference?

Reply:
Done